wdl-grammar/src/v1/lint/missing_runtime_block.rs

```diff
@@ -0,0 +1,270 @@
+//! workflow tasks _must_ have a `runtime` block to ensure portability.
+
+use std::collections::VecDeque;
+
+use nonempty::NonEmpty;
+use pest::iterators::Pair;
+use wdl_core::concern::code;
+use wdl_core::concern::lint;
+use wdl_core::concern::lint::Group;
+use wdl_core::concern::lint::Rule;
+use wdl_core::concern::Code;
+use wdl_core::file::Location;
+use wdl_core::Version;
+
+use crate::v1;
+
+/// Every task _must_ have a `runtime` block.
+#[derive(Debug)]
+pub struct MissingRuntimeBlock;
+
+impl<'a> MissingRuntimeBlock {
+    /// Creates an error corresponding to the task with a missing `runtime`
+    /// block.
+    fn missing_runtime_block(&self, location: Location) -> lint::Warning
+    where
+        Self: Rule<&'a Pair<'a, v1::Rule>>,
+    {
+        // SAFETY: this error is written so that it will always unwrap.
+        lint::warning::Builder::default()
+            .code(self.code())
+            .level(lint::Level::Medium)
+            .group(self.group())
+            .push_location(location)
+            .subject("missing runtime block")
```

Comment:
```suggestion
            .subject("missing runtime block for task")
```

I might just clarify here that it's a runtime block _for a task_.

Reply:
This would also necessitate the tests being updated to match this.

Reply:
Done, I have made a new commit with updates to tests and gauntlet.toml

Reply:
Done!